sbi/simulators/simutils.py

```diff
@@ -5,14 +5,62 @@
 import torch
 from torch import Tensor
 from tqdm.auto import tqdm
+from sbi.simulators.mp_simulator import simulate_mp
+
+
+def simulate_mp_in_batches(
+    simulator: Callable,
+    theta: Tensor,
+    sim_batch_size: Optional[int],
+    number_of_workers: int = 4,
+    worker_batch_size: int = 20,
+    show_progressbar: Optional[bool] = True,
+):
+    """
+    Distributes all parameters theta to simulate_in_batches, i.e. splits them between
+     cores and then processes them in batches.
+
+    Args:
+        simulator: simulator function.
+        theta: parameters $\theta$ sampled from prior or posterior.
+        number_of_workers: how many parallel workers to start
+        worker_batch_size: how many params are processed on each worker. This number is
+            thetas will then be handled by simulate_in_batches()
+        sim_batch_size: number of simulations per batch. Default is to simulate
+            the entire theta in a single batch.
+        show_progressbar: whether to show a progressbar during simulating
+
+    Returns: parameters theta and simulation outputs x
+
+    """
+
+    assert worker_batch_size >= sim_batch_size, (
+        "worker_batch_size has to be smaller than simulation_batch_size when using"
```

Comment:
Error message doesnt match the criterion.

Reply:
this asserts that worker batchs size is larger or equal sim batch size. but the message says the opposite. if the criterion is correct, then the message shoud say: 
`worker_batch_size must not be smaller than simulation_batch_size when using multiprocessing`

Or am I missing something?